src/runtimes/docker/contracts/ContainerClient.ts

```diff
@@ -609,6 +614,30 @@ type InspectImagesCommand = {
     inspectImages(options: InspectImagesCommandOptions): Promise<PromiseCommandResponse<Array<InspectImagesItem>>>;
 };
 
+export type ImageSbomItem = {
+    /**
+     * The RAW inspect output
+     */
+    raw: string;
+};
+/**
+ * Options for inspecting images
+ */
+export type ImageGenerateSbomCommandOptions = CommonCommandOptions & {
+    /**
+     * The image names/IDs/etc. to inspect, passed directly to the CLI
+     */
+    imageRef: string;
+};
+
+type ImageGenerateSbomCommand = {
```

Comment:
SbomForImageCommand to stay consistent with the naming convention for the rest of the commands.

Reply:
Yeah that make more sense!